src/pages/Home/Banner/index.module.scss

```diff
@@ -5,7 +5,7 @@ $backgroundColor: #232323;
 .root,
 .fiberBanner {
   width: 100%;
-  height: 200px;
+  height: 300px;
```

Comment:
Extra vertical padding is introduced by this
![image](https://github.com/user-attachments/assets/2dcab126-559b-4ece-9b8d-bb7fdc2b5741)
![image](https://github.com/user-attachments/assets/51e53e9e-7475-4a7a-81ef-da542eace071)